xclim/indicators/atmos/_synoptic.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""Synoptic indicator definitions."""
+
+from xclim import indices
+from xclim.core.indicator import Daily
+
+__all__ = ["jetstream_metric_woolings"]
+
+
+class JetStream(Daily):
+    """Indicator involving daily u- and/or v-component wind series."""
```

Comment:
I agree that this is confusing. In `xclim.core.indicator` we define multiple base `Indicator` classes. The most basic is `Indicator` and its most used child `ResamplingIndicator`. The latter is meant for indicators that resample along the `time` dimension, those that have a `freq` argument. Because that is the case of most indicators, for convenience and also for "historical" reasons (not wanting to make breaking changes), there are a few subclasses that inherit from `ResamplingIndicator` even if their name doesn't explicitly show it. It is the case of `Daily` which adds the condition that inputs have a daily frequency.

In your case, the `jetstream_metric_woolings` is _not_ resampling, so you'll want to use the `Indicator` base class directly. To have the daily input check, you can add `src_freq='D'` (default is `None` which deactivates any frequency checks).

```suggestion
from xclim.core.indicator import Indicator

__all__ = ["jetstream_metric_woolings"]


class JetStream(Indicator):
    """Indicator involving daily u- and/or v-component wind series."""
    
    src_freq = 'D'
```